interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -14,7 +14,16 @@ angular.module('adage.analyze.sampleBin', [
 
 .factory('SignatureSet', ['$resource', 'ApiBasePath',
   function($resource, ApiBasePath) {
-    return $resource(ApiBasePath + 'node/set/:ids/');
+    return $resource(
+      ApiBasePath + 'node/post_multiple/',
+      {},
+      {post: {
+        method: 'POST',
+        // Setting Content-Type is required. Django will not process the
+        // POST data the way Angular's defaults send it.
+        headers: {'Content-Type': 'application/x-www-form-urlencoded'}
```

Comment:
It was a good idea, but didn't work out. I've reverted the implementation to this `'text/plain'` encoding. Filing an issue to sort out the testing glitch that this causes as a separate PR.